cmake/Caffe2Config.cmake.in

```diff
@@ -100,6 +100,13 @@ if(@USE_XPU@)
   list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}")
   include("${CMAKE_CURRENT_LIST_DIR}/public/xpu.cmake")
   set(CMAKE_MODULE_PATH "${old_CMAKE_MODULE_PATH}")
+
+  if(@USE_XPU@ AND NOT PYTORCH_FOUND_XPU)
+    message(FATAL_ERROR
```

Comment:
This message is similar to the one we have for cuda above:
https://github.com/pytorch/pytorch/blob/d25617255c9d9a9a8c3104f9c854ef581f02c50c/cmake/Caffe2Config.cmake.in#L89-L93